viscy/data/hcs.py

```diff
@@ -104,6 +104,8 @@ class SlidingWindowDataset(Dataset):
     :param ChannelMap channels: source and target channel names,
         e.g. ``{'source': 'Phase', 'target': ['Nuclei', 'Membrane']}``
     :param int z_window_size: Z window size of the 2.5D U-Net, 1 for 2D
+    :param int pyramid_resolution: pyramid level.
```

Comment:
Outdated type hint. Also I'm wondering if this is the best name, since it's valid for any multi-array FOV group that may or may not be a pyramid.

Reply:
Do you have any name suggestions? I can add in the docstring that "0" is (highest-resolution). 
This parameter is also totally optional. It's defaulted to `"0"`, and only if you have a multiscale you would probably look for this parameter.

Reply:
Yes for now explaining in docstring is sufficient.